pkgs/development/compilers/adoptopenjdk-bin/generate-sources.py

```diff
@@ -15,6 +15,8 @@
     "x64": ("x86_64",),
     "aarch64": ("aarch64",),
     "arm": ("armv6l", "armv7l"),
+    "ppc64": ("powerpc64",),
```

Comment:
Is there any reference to this? I could only find powerpc64le.

Reply:
Removed.